src/interface/src/app/map/map.component.ts

```diff
@@ -66,7 +95,37 @@ export class MapComponent implements AfterViewInit {
     zoomControl.addTo(this.map);
   }
 
-  // Renders the existing project boundaries + metadata in a popup in an optional layer.
+  /** Gets the selected region geojson and renders it on the map. */
+  private displayRegionBoundary(selectedRegion: Region | null) {
+    console.log({selectedRegion});
+    if (selectedRegion) {
+      const path = regionToGeojsonMap[selectedRegion];
+      if (!path) {
+        return;
+      }
+      // Type 'any' is used in order to access coordinates
+      this.http.get(path).subscribe((boundary: any) => {
```

Comment:
I think the general recommendation is to make HTTP calls from services only. Can we add this to the MapService?

Reply:
yes good catch! I was planning on moving it to mapService. 
to answer the question about .geojson - yeah, I meant to check it in because it's much faster to load it locally than from the server.

Reply:
Done!